low-level/files/opt/seagate/sspl/bin/sspl_config

```diff
@@ -185,15 +185,28 @@ case $cmd in
         ;;
 esac
 
+rabbitmq_version=$(rpm -qi rabbitmq-server | awk -F': ' '/Version/ {print $2}')
```

Comment:
> major_release . minor_release . maintenance_release

use this nomenclature below so its relatable easily

Reply:
Ok